bigchaindb_driver/connection.py

```diff
@@ -25,36 +31,79 @@ def __init__(self, *, node_url, headers=None):
         if headers:
             self.session.headers.update(headers)
 
-    def request(self, method, timeout=None, *, path=None, json=None,
-                params=None, headers=None, **kwargs):
+        self._retries = 0
+        self.backoff_time = None
+
+    def request(self, method, *, path=None, json=None,
+                params=None, headers=None, timeout=None, **kwargs):
         """Performs an HTTP requests for the specified arguments.
 
+           If the node has backoff time attached, waits till the time is due.
+
+           Backoff time is adjusted after the request is made.
```

Comment:
Can you go a bit more into detail with the comment here? In the beginning it was a bit confusing for me. I also didn't get why there was no `backoff_time` in the function.

Reply:
sorry for commenting twice, I thought I forgot to do it last week :D